pkgs/top-level/php-packages.nix

```diff
@@ -59,6 +59,72 @@ lib.makeScope pkgs.newScope (self: with self; {
     pname = "php-${pname}";
   });
 
+  # Function to build a single php extension based on the php version.
```

Comment:
Maybe mention that this function is internal and that it builds extensions shipped as part of the PHP project.

We might even call it `mkBundledExtension` or something.

Reply:
I updated the documentation to this:
```
  # Function to build an extension which is shipped as part of the php
  # source, based on the php version.
```

Personally, I think `mkBundledExtension` is a bit of a mouthful, so I left it as `mkExtension`. I don't care too much, though, so if anyone prefers `mkBundledExtension` let me know and I'll rename it.